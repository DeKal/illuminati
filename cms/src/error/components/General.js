import React from 'react'
import PropTypes from 'prop-types'
import Layout from 'error/components/base/Layout'

const General = ({ message }) => {
  return (
    <Layout
      title={'An error occurred'}
      description={
        'There are some errors happened during the process, please back to home page and try again.'
      }
      detail={message}
    />
  )
}

export default General

General.propTypes = {
  message: PropTypes.string
}
