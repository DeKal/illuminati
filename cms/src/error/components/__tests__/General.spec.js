import * as React from 'react'
import { shallow } from 'enzyme'
import General from '../General'

describe('General', () => {
  it('should render genral error page correctly', function() {
    const wrap = shallow(<General message="Error from script abc" />)
    expect(wrap).toMatchSnapshot()
  })
})
