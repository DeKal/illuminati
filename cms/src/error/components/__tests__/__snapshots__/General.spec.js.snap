// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`General should render genral error page correctly 1`] = `
<Layout
  description="There are some errors happened during the process, please back to home page and try again."
  detail="Error from script abc"
  title="An error occurred"
/>
`;
